integration/README.md

```diff
@@ -3,45 +3,70 @@
 
 Integration tests for Cook scheduler
 
-To run the tests on a local cook install, run
+## Running in docker
+
+If you're running cook in docker, it can be beneficial to run the integration tests in a docker container so that the IP addresses resolve correctly. To do so, there are two separate scripts:
```

Comment:
I would favor one sentence per line in markdown.

Reply:
I'm going to un-move these sections to de-clutter this PR. We can open a new PR later to fix the flow (and the formatting) of the README.